src/screens/TripDetails/DepartureDetails/index.tsx

```diff
@@ -67,6 +67,7 @@ export default function DepartureDetails({navigation, route}: Props) {
   const [activeItemIndexState, setActiveItem] = useState(activeItemIndex);
   const {theme} = useTheme();
   const {modesWeSellTicketsFor} = useFirestoreConfiguration();
+  const {enable_ticketing} = useRemoteConfig();
```

Comment:
This is set as default `true` now, yes?

Reply:
Seems to be, yes. 

https://github.com/AtB-AS/mittatb-app/blob/abbc8d71bf8497279f1187ec0aa2df291b1d8e79/src/remote-config.ts#L56